packages/debug/src/browser/view/debug-configuration-widget.tsx

```diff
@@ -51,6 +51,11 @@ export class DebugConfigurationWidget extends ReactWidget {
         this.toDispose.push(this.manager.onDidChange(() => this.update()));
         this.toDispose.push(this.workspaceService.onWorkspaceChanged(() => this.update()));
         this.toDispose.push(this.workspaceService.onWorkspaceLocationChanged(() => this.update()));
+        this.scrollOptions = {
```

Comment:
thanks @xieyuanhuata for fixing this issue! 

I'm generally fine with disabling perfect scroll bar as as, but why should it be created in the first place? 
I think unsetting it in `ctor` like this would also to the job. @akosyakov is there a use for the scrollbar at all in those two widgets?
```
    constructor() {
        super();
        this.scrollOptions = undefined;
    }
```

Reply:
@AlexTugarev I agree with your opinion

Reply:
no, unsetting like @AlexTugarev suggested should work as well

Reply:
@AlexTugarev  @akosyakov
you are right! Unsetting like @AlexTugarev suggested work as well.